src/onboarding2/Create/Create.js

```diff
@@ -0,0 +1,260 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react'
+import { useViewport, GU } from '@aragon/ui'
+import templates from '../../templates'
+import Templates from '../Templates/Templates'
+import CreateStepsPanel from './CreateStepsPanel'
+import { loadTemplateState, saveTemplateState } from '../create-utils'
+
+const STATUS_SELECT_TEMPLATE = Symbol('STATUS_TEMPLATE')
+const STATUS_TEMPLATE_SCREENS = Symbol('STATUS_TEMPLATE_SCREENS')
+const STATUS_LAUNCH = Symbol('STATUS_LAUNCH')
+
+// Used during the template selection phase, since we don’t know yet what are
+// going to be the configuration steps.
+const PLACEHOLDER_SCREENS = [
+  'Claim domain',
+  'Configure template',
+  'Review information',
+]
+
+function getSteps(status, template, templateData) {
+  if (!template || status === STATUS_SELECT_TEMPLATE) {
+    return ['Select template', ...PLACEHOLDER_SCREENS, 'Launch organization']
+  }
+  return [
+    template.name,
+    ...((template.template && template.template.screens) || []).map(([name]) =>
+      typeof name === 'function' ? name(templateData) : name
+    ),
+    'Launch organization',
+  ]
+}
+
+// Handle and store everything related to a template state.
+function useTemplateState() {
+  const [templateId, setTemplateId] = useState(null)
+  const [templateScreenIndex, setTemplateScreenIndex] = useState(-1)
+  const [templateData, setTemplateData] = useState({})
+
+  useEffect(() => {
+    if (templateId) {
+      saveTemplateState({
+        templateData,
+        templateId,
+        templateScreenIndex,
+      })
+    }
+  }, [templateData, templateId, templateScreenIndex])
+
+  useEffect(() => {
+    const {
+      templateData,
+      templateId,
+      templateScreenIndex,
+    } = loadTemplateState()
+
+    if (templateId) {
+      setTemplateData(templateData)
+      setTemplateId(templateId)
+      setTemplateScreenIndex(templateScreenIndex)
```

Comment:
cc @dizzypaty We should probably check for this in the initial onboarding screen and prompt the user if they'd like to continue their last creation attempt.

If they land directly on `/create`, it would also be nice to show a modal or etc. to ask if they'd like to jump back.